package.json

```diff
@@ -11,7 +11,7 @@
   },
   "main": "./dist/vertical.js",
   "scripts": {
-    "deploy": "gh-pages -t -d gh-pages -m \"Build for $(git log --pretty=format:%H -n1)\"",
+    "deploy": "rimraf gh-pages/closure-library/scripts/ci/CloseAdobeDialog.exe && gh-pages -t -d gh-pages -m \"Build for $(git log --pretty=format:%H -n1)\"",
```

Comment:
🐃  🐴  :wat: :wat: :wat: :wat: :wat: :wat: :wat:

Reply:
`rimraf` is a cross-platform utility for deleting things in Node, basically so that `deploy` will continue to work on Windows.

Reply:
Ah yeah that was totally the part I was confused about 🥇